governance/docs/what_is_governance.md

```diff
@@ -0,0 +1,61 @@
+
+A well-designed system of governance is much less likely to turn away or de-motivate project participants than a vague or non-existent one is. Consider your project from the perspective of new contributors: are they more or less likely to jump into a project without any sense of the role they're supposed to play, and the rules they're supposed to follow, when they want project leaders to seriously consider their contributions? A clear governance model helps people understand precisely how they can make an immediate contribution to a project, how they can pitch in without "rocking the boat," how they can escalate questions or issues if they have them, and what sorts of leadership positions they can aspire to if they stick around long enough.
+
+A community's goal in designing a governance model should be making structures of participation obvious. When your project's rules are clear, contributors can engage with confidence. Taking this approach to governance can positively impact a project's long-term viability and growth.
+
+## Governance is People
+
+Simply put, "governance" refers to the rules or customs that determine who gets to do what (or is supposed to do what), how they're supposed to do it, and when.
+
+All of project governance falls into two categories: roles, or policies and procedures. For the purpose of explanation, we'll discuss each of these separately. In practice, however, they're inseparable—two sides of the same coin.
+
+## Roles
+
+A great deal of activity hinges on roles-related governance in open source projects. Think of a role as a function someone in the project performs.  Project contributors can, and often do, have multiple roles, and project roles are often fulfilled by multiple people.
+
+As a part of governance, all projects have roles, even if they are not explicitly documented.  Documenting them will help you recruit people to those roles. A minimal set of project roles will often be something like: contributor, reviewer, maintainer.
+
+When documenting your project’s roles, ask the following questions:
+
+* What roles do (or can) project contributors play?
+* What qualifies a person to play a particular role in the project?
+* What duties, privileges, and forms of authority are associated with each role?
+* What project resources are the province or responsibility of people who perform certain roles?
+
+See the [section on Roles]() for more detail and examples of how to develop and document them for your project.
+
+## Policies and Procedures
+
+While role definitions explain how specific contributors participate in the project, there are many activities in a project that involve groups of people, including people external to the project.  Examples of this include things like a release procedure or a contribution process.  These Policies and Procedures (P&P) are what is often thought of as “governance paperwork” for projects.
```

Comment:
This is the only place where I see typographer's quotes in the document...

Reply:
and ... ?

Reply:
Sorry, it's just a nit. Just makes the look of the rendered markdown inconsistent, at least as GitHub renders it.

"governance" gets a bad rap
...
“governance paperwork”

Reply:
I don't see anything to do here.